adapters/deepintent/deepintent.go

```diff
@@ -21,7 +21,7 @@ type DeepintentAdapter struct {
 }
 
 type deepintentParams struct {
-	TagID string `json:"tagId"`
+	tagId string `json:"tagId"`
```

Comment:
This change isn't really necessary. I'm fine if you want to keep it this way or how it was before.

The capitalization of the first character of a field in Go is the access modifier. Upper case = exported for other packages. Lower case = package private. Since the struct is already package private, the access modifier doesn't really matter.

The json serializer will use the name `tagId` from the json attribute.

Reply:
This is changed to keep in sync the standard names in adapter. I think this should be fine for now.